workspace-manager-clienttests/src/main/java/scripts/testscripts/CreateGetDeleteDataReference.java

```diff
@@ -0,0 +1,60 @@
+package scripts.testscripts;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiException;
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import bio.terra.workspace.model.CreateDataReferenceRequestBody;
+import bio.terra.workspace.model.CreateWorkspaceRequestBody;
+import bio.terra.workspace.model.CreatedWorkspace;
+import bio.terra.workspace.model.DataReferenceDescription;
+import bio.terra.workspace.model.DataRepoSnapshot;
+import bio.terra.workspace.model.ReferenceTypeEnum;
+import bio.terra.workspace.model.WorkspaceDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.DataReferenceUtils;
+import scripts.utils.WorkspaceManagerServiceUtils;
+
+import java.util.List;
+import java.util.UUID;
+import scripts.utils.WorkspaceTestScriptBase;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class CreateGetDeleteDataReference extends WorkspaceTestScriptBase {
+  private static final Logger logger = LoggerFactory.getLogger(CreateGetDeleteDataReference.class);
+  private UUID workspaceId;
+
+  @Override
+  public void doSetup(List<TestUserSpecification> testUsers, WorkspaceApi workspaceApi) throws ApiException {
+    workspaceId = UUID.randomUUID();
+    CreateWorkspaceRequestBody requestBody = new CreateWorkspaceRequestBody().id(workspaceId);
+    workspaceApi.createWorkspace(requestBody);
+    WorkspaceManagerServiceUtils.assertHttpOk(workspaceApi, "CREATE workspace");
```

Comment:
I think right now the base class does this for you. It's a bit magical though.

Reply:
Which part? if it's calling `assertHttpSuccess`, I removed that from the base class as it caused empty setups/cleanups to fail.

Reply:
Yes, killing that in the base class is the way to go. It also prevented calling get after delete to verify a 404.